index.mjs

```diff
@@ -57,6 +60,9 @@ import yaml from 'js-yaml';
  *   built-ins/Temporal/Duration/compare/argument-string-negative-fractional-units.js
  *   built-ins/Temporal/Duration/from/argument-string-negative-fractional-units.js
  *   ```
+ * @property {boolean=} updateExpectedFailureFiles Whether to edit
+ *   expected-failure files on-disk to reflect the status of the tests as reported
+ *   in this run.
```

Comment:
Two things: 

1. I think this text could be clarified, esp. to explain what the editing does (remove expected passes) and doesn't do. (create files for new failures). Perhaps use language like this?

> Optionally removes passing tests from expected-failure files on disk. Used in local development to automatically revise expected-failure files after making code changes that fix test failures. Do not use this option in CI.

2. Should update the README file to add the same docs there too.